.github/mergify.yml

```diff
@@ -1,11 +1,28 @@
 ---
 defaults:
   actions:
-    merge:
+    queue:
       method: rebase
       rebase_fallback: merge
-      strict: smart
-      strict_method: rebase
+      update_method: rebase
+queue_rules:
+  - name: default
+    conditions:
+      # Conditions to get out of the queue (= merged)
+      - base~=^(main)|(release-.+)$
```

Comment:
You do not need to include the branch name here. Only PRs that get the `queue` action will land here. The same counts for the `approved-reviews-by`/`changes-requested` checks.

I think you only need the `status-success` checks here.